### PR TITLE
Fix/backend20: * add templates to project build and fix their load path

### PR DESCRIPTION
**Pull request by shirokuronemuri**
I found out that the email templates weren't copied to the build folder on project build.

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "prestart": "npm run migrate:prod && npm run seed:soft",
     "start": "node ./build/server.js",
     "build:ts": "tsc && tsc-alias -p tsconfig.json",
-    "build": "npm run build:ts"
+    "build": "npm run build:ts && cp -r src/shared/mail-templates build/shared/mail-templates"
```
Comment (VitaliiBedletskyi) [inline backend/package.json]: It seems better to execute this command during building the shared package. Also, could you please create a js file (`build.js`) and implement copy logic in the js file using the `fs` module from `nodejs`?

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

File: backend/package.json (hunk 1 above)
Comment (ihor-pavliuk) [reply backend/package.json:20]: so should I move mail templates from backend to shared folder as well?

File: backend/package.json [hunk 2]
```diff
@@ -46,6 +47,7 @@
     "cloudinary": "1.30.1",
     "dotenv": "16.0.1",
     "email-templates": "10.0.1",
+    "fs-extra": "^10.1.0",
```
Comment (VitaliiBedletskyi) [inline backend/package.json]: Why do you need to use fs-extra instead of fs module from nodejs?

Comment (VitaliiBedletskyi) [reply backend/package.json]: B19: The versions of dependencies used are fixed in package.json.

**Review by VitaliiBedletskyi — APPROVED** [review]

**Pull request merged**